demosplan/DemosPlanCoreBundle/Repository/ProcedureRepository.php

```diff
@@ -1319,25 +1341,30 @@ private function validateProcedureTemplate(Procedure $procedure, string ...$addi
      * The result will not contain deleted procedures.
      *
      * @return array<int, Procedure>
+     *
+     * @throws PaginationException
+     * @throws PathException
+     * @throws SortException
      */
     public function getProceduresReadyToSwitchPhases(): array
     {
         $query = $this->createFluentQuery();
-        $orCondition = $query->getConditionDefinition()
+        $conditionDefinition = $query->getConditionDefinition()
             ->propertyHasValue(false, ['deleted'])
             ->propertyHasValue(false, ['master'])
-            ->propertyHasValue(false, ['masterTemplate'])
-            ->anyConditionApplies();
+            ->propertyHasValue(false, ['masterTemplate']);
+
+        $orCondition = $conditionDefinition->anyConditionApplies();
```

Comment:
why did you pull this into an own line?

Reply:
This is intended because otherwise the reading is misleading -> this way I tried to improve the readability.
In short: Actually "anyconditionApplies()" is a kind of opening bracket and the following conditions will be in it